webapp/src/pages/eventConfig/components/EventConfigComponent.js

```diff
@@ -414,22 +699,33 @@ class EventConfigComponent extends Component {
 
           <div className={"form-group row"}>
             <div className={"col-sm-4 ml-md-auto"}>
-              <button
-                className="btn btn-danger btn-lg btn-block"
-                onClick={() => this.onClickCancel()} >
+              <Link to=".." className="btn btn-danger btn-lg btn-block">
                 {t("Cancel")}
-              </button>
+              </Link>
             </div>
-
+            
             <div className={"col-sm-4 "}>
-              <button
-                onClick={() => this.onClickSubmit()}
-                className="btn btn-success btn-lg btn-block"
-                disabled={!hasBeenUpdated}>
-                {t("Update Event")}
-              </button>
+              {isNewEvent ? (
+                <Link to="..">
```

Comment:
Ah sorry for not being more precise when I told you about Link! It will work fine for your cancel button, but for your submit you'll only want to go back conditionally if the post succeeds and if there are no validation or post errors, so Link won't work for this because i think it will always go back no matter what happens in your onClickSubmit function (is that what you're observing?) so for this case, you can put some code in your onClickSubmit after succeeding that navigates back instead, for example: `this.props.history.push('/')` to go back to the organisation home page.